### DISCUSSION TITLE
Store Array Multidimensional Not Working in Laravel Livewire

### DISCUSSION THREAD

**Discussion by gehand01** [livewire/livewire#5434] [1. Help]
Hi, Dev! I'm having trouble handling store using Array in Laravel livewire, with the same text input variable.

My Blade This is selected Input

<select class="form-control" id="controlOnes">
    <option>-- Choose One --</option>   
        @foreach ($dataBiodata as $listBiodata)
            <option 
                data-fullname='{{ $listBiodata->fullname }}'
                data-account_number='{{ $listBiodata->account_number }}'
                data-gender='{{ $listBiodata->gender }}'
                data-birth_place='{{ $listBiodata->birth_place }}'
                value="{{ $listBiodata->fullname}}">
                     {{ $listBiodata->fullname }}
            </option>
         @endforeach
</select>

<form wire:submit.prevent="save">
     @csrf
       <div class="col-md-12">
           @foreach ($dataDocument as $index => $listDocument)
               @if ($listDocument->filesIds == $letter->idsFile)
               <div>
               <input type="text" id="{{ $listDocument->category_component }}" wire:model="dataDocument.{{ $index }}.desc_component" class="form-control mb-3">
               </div>
               @endif
            @endforeach
            <hr />
        <button class="btn btn-success" type="submit">Store</button>
       </div>
</form>

My Script ( JavaScript )

$().ready(function() {
        $("#controlOnes").change( function() {
            let fullname = $(this).find("option:selected").data("fullname")
            let account_number = $(this).find("option:selected").data("account_number")
            let gender = $(this).find("option:selected").data("gender")
            let birth_place = $(this).find("option:selected").data("birth_place")
            $("#fullname").val(fullname)
            $("#account_number").val(account_number)
            $("#gender").val(gender)
            $("#birth_place").val(birth_place)
        })
    })

And My Component Livewire

public $letter, $code_component, $dataDocument;
    
    public function mount($idFiles)
    {
        $this->letter = FileModel::find($idFiles);
    }

    public function render()
    {
        $this->dataDocument = DB::table('documents_files')
            ->join('documents_files_additional','documents_files.idsFile','=','documents_files_additional.filesIds')
            ->join('documents_folder','documents_files.file_codes','=','documents_folder.folder_codes')
            ->select('documents_files.*','documents_files_additional.*','documents_folder.*')
            ->where('documents_files.code_ids', Auth::user()->code_access)
            ->get();
        $this->dataBiodata = DB::table('biodata')
            ->where('family_number', Auth::user()->family_number)
            ->get();
        return view('livewire.backend.documents.letter-component', [
            'letter' => $this->letter
        ]);
    }

    public function save()
    {
        ??? My Brain Is Splitting :( 
    }

Maybe your ideas can help me with my problem

**Comment by joshhanley** [comment]
@gedehandikaWebProgrammer your code formatting seems to not be loading properly, which is making it hard to follow. Can you fix it up please? 

When you paste code into github, you need to surround it in triple back ticks and then specify the file type like php or blade.

For example
\```php
\\ PHP code here
\```

\```blade
\\ HTML and blade code here
\```

Thanks!
